pkgs/development/python-modules/okta/default.nix

```diff
@@ -20,20 +20,30 @@
 , yarl
 }:
 
+let
+  aenum_fixed = aenum.overridePythonAttrs (oldAttrs: rec {
```

Comment:
Overriding in `python-modules` should not be done. The proper fix would be to get support for `aenum` > 3.1.11 in `okta`.

https://github.com/okta/okta-sdk-python/issues/368

Reply:
I took the initiative to make a PR that would fix their package https://github.com/okta/okta-sdk-python/pull/370